stellargraph/layer/appnp.py

```diff
@@ -362,14 +362,7 @@ def __call__(self, x):
 
         return h_layer
 
-    def in_out_tensors(self, multiplicity=None):
-        """
-        Builds an APPNP model for node or link prediction
-
-        Returns:
-            tuple: `(x_inp, x_out)`, where `x_inp` is a list of Keras/TensorFlow
-            input tensors for the model and `x_out` is a tensor of the model output.
-        """
+    def _tensors(self, multiplicity, extra_layers):
```

Comment:
Im not sure if I'm missing someting, but `extra_layers` doesn't seem to be used here so I think this is always just creating a regular APPNP model

Reply:
Oh, oops. Missed the loop.